workflow-Estimate/src/main/java/ca/on/oicr/pde/workflows/EstimateWorkflow.java

```diff
@@ -0,0 +1,247 @@
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public void setupDirectory() {
+        init();
+        this.addDirectory(dataDir);
+        this.addDirectory(tmpDir);
+        if (!dataDir.endsWith("/")) {
+            dataDir += "/";
+        }
+        if (!tmpDir.endsWith("/")) {
+            tmpDir += "/";
+        }
+    }
+
+    @Override
+    public Map<String, SqwFile> setupFiles() {
+        /**
+         * Provisioning multiple RSEM files
+         */
+        Map<String,List<String>> inputFileMap = this.getRsemStarMap(this.inputRSEMFiles, this.inputSTARFiles);
+        for (String key : inputFileMap.keySet()){
+            SqwFile file0 = this.createFile("RSEM_"+key);
+            SqwFile file1 = this.createFile("STAR_"+key);
+            String rsemFile = inputFileMap.get(key).get(0);
+            String starFile = inputFileMap.get(key).get(1);
+            file0.setSourcePath(rsemFile);
+            file0.setType(TXT_METATYPE);
+            file0.setIsInput(true);
+            file1.setSourcePath(starFile);
+            file1.setType(TXT_METATYPE);
+            file1.setIsInput(true);
+        }  
+        return this.getFiles();
+    }
+
+    @Override
+    public void buildWorkflow() {
+        Job parentJob = null;
+        String postProcessedRSEM = this.dataDir + this.outputFilenamePrefix + "_genes_all_samples_RCOUNT.txt";
+        
+        String estimateGCT = postProcessedRSEM + ".estimate.gct";
+        String ssGSEA = postProcessedRSEM + ".ssGSEA.txt";
+        
+        Job preProcess = postProcessRSEM(this.inputRSEMFiles, this.inputSTARFiles, postProcessedRSEM);
+        parentJob = preProcess;
+        
+        Job runEstimate = launchEstimate(postProcessedRSEM);
+        runEstimate.addParent(parentJob);
+        parentJob = runEstimate;
+        
+        // Provision out estimae and ssgsea outputs
+        SqwFile estimateOutput = createOutputFile(estimateGCT, TXT_METATYPE, this.manualOutput);
+        estimateOutput.getAnnotations().put("estimate_output", "estimate");
+        parentJob.addFile(estimateOutput);
+        
+        SqwFile ssGSEAOutput = createOutputFile(ssGSEA, TXT_METATYPE, this.manualOutput);
+        ssGSEAOutput.getAnnotations().put("ssgsea_output", "ssgsea");
+        parentJob.addFile(ssGSEAOutput);
+//        
+    }
+    
+    private Job launchEstimate(String inRSEM) {
+        Job runEst = getWorkflow().createBashJob("run_estimate_ssgsea");
+        Command cmd = runEst.getCommand();
+        cmd.addArgument("export R_LIBS=" + rLib + ";");
+        cmd.addArgument(this.rScript);
+        cmd.addArgument(this.estimateScript);
+        cmd.addArgument(inRSEM);
+        cmd.addArgument(this.dataDir);
+        cmd.addArgument(this.gmtFile);
+        cmd.addArgument(this.ensFile);
+        cmd.addArgument(getWorkflowBaseDir() + "/dependencies/convert_rsem_results_zscore.r");
+        runEst.setMaxMemory(Integer.toString(this.estimateMem * 1024));
```

Comment:
You already get this value up above at line 97, so you should use it from there.